CMakeLists.txt

```diff
@@ -145,6 +145,8 @@ if(BUILD_TOXAV)
   endif()
 endif()
 
+add_definitions(-DCMP_NO_FLOAT=1)
```

Comment:
Please add a  comment what this does/why it's needed.

Reply:
Done.